### DISCUSSION TITLE
support miniMapNode  handle click event by user side

### DISCUSSION THREAD

**Discussion by Fiv5** [xyflow/xyflow#1343] [New Features]
Sometimes we want to handle something when clicking MiniMapNode.

**Comment by Fiv5** [comment]
I submit a PR in https://github.com/wbkd/react-flow/pull/1344

**Comment by kosuke-suzuki** [comment]
@moklick 
Could you give a comment?

This centering-on-click feature on miniMap is really appreciated.
I believe this feature is on the way to feature request #236 

We also gave some tries to @Fiv5 #1344 , which seemed promising.

**Comment by moklick** [comment]
locked in favour of locked in favour of #2166
